src/HasStatuses.php

```diff
@@ -46,29 +54,36 @@ public function isValidStatus(string $name, string $reason = ''): bool
      */
     public function latestStatus(...$names): ?Status
     {
-        $names = is_array($names) ? array_flatten($names) : func_get_args();
+        $names = \is_array($names) ? array_flatten($names) : \func_get_args();
```

Comment:
Remove `/`. We are sure we don't have that function in the namespace.

Reply:
The `\` added before those functions are suggested by PHPStorm (PHP EA Inspections plugin)

See also: https://github.com/Roave/FunctionFQNReplacer 

If you are aware of that (and pretty sure you probably are), I will remove that too.

Reply:
This (https://github.com/symfony/symfony/commit/bccab35aa2c0a6500281083414346a0417c234ac) commit from Symfony would be more convincing =)